src/continuum_opacity/opacity_H.jl

```diff
@@ -183,10 +186,10 @@ function Hminus_ff(nH_I_div_partition::Flt, ne::Flt, ν::Flt, ρ::Flt,
     # Pₑ * α_ff(H⁻) gives the absorption coefficient in units of cm² per ground state H I atom
     Pₑ = ne * kboltz_cgs * T
 
-    # in this case, I'm going to account for the fact that n(H I, n=1) might be slightly
-    # smaller than the entire number density of H I. This is only a difference at the highest
-    # temperatures. For the temperature range where this approximation is valid, less that
-    # 0.23% of all H I atoms are not in the ground state.
+    # Going to account for the fact that n(H I, n=1) might be slightly smaller than the entire
+    # number density of H I. This is only a difference at the highest temperatures. For the
+    # temperature range where this approximation is valid, less than 0.23% of all H I atoms are not
+    # in the ground state.
```

Comment:
Is there a typo in the first sentence here?

Reply:
Yeah, you're totally right. Fixed.